__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("app", () => {
+  describe("GET /api/topics", () => {
+    test("status:200 responds with an array of all topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          console.log(response.body, "response");
+          const { topics } = response.body;
+          expect(topics).toBeInstanceOf(Array);
+          topics.forEach((topic) => {
```

Comment:
As you're using forEach, could you just pop another expect statement in beforehand to check the length of the array please?